delphi-frontend/src/main/antlr3/au/com/integradev/delphi/antlr/Delphi.g

```diff
@@ -687,12 +689,13 @@ attributeList                : attributeGroup+
 attributeGroup               : lbrack (attribute ','?)+ rbrack
                              -> ^(TkAttributeGroup<AttributeGroupNodeImpl> attribute+)
                              ;
-attribute                    : nameReference argumentList?
-                             -> ^(TkAttribute<AttributeNodeImpl> nameReference argumentList?)
+attribute                    : (ASSEMBLY ':')? nameReference argumentList? (':' nameReference argumentList?)*
+                             -> ^(TkAttribute<AttributeNodeImpl> ASSEMBLY? nameReference argumentList? (':' nameReference argumentList?)*)
                              ;
 
 //----------------------------------------------------------------------------
 // Expressions
+// Expressions
```

Comment:
This line looks like it was accidentally duplicated.

Reply:
Weird, no idea how that happened.